index.js

```diff
@@ -0,0 +1 @@
+console.log("Hello World!");
```

Comment:
I think we can change this World to Katja so it will be more defined. What do you think?

Reply:
Thanks for catching, I updated it!